spec/sample_manifest_excel/upload/upload_spec.rb

```diff
@@ -332,21 +330,26 @@
 
     context 'plate' do
       let!(:plate_columns) { SampleManifestExcel.configuration.columns.plate_full.dup }
-      let!(:download) { build(:test_download_plates, columns: plate_columns) }
+      let(:download) { build(:test_download_plates, columns: plate_columns, study: study) }
+      let(:study) { create(:open_study, accession_number: 'acc') }
+      let(:upload) { SampleManifestExcel::Upload::Base.new(file: test_file, column_list: plate_columns, start_row: 9) }
 
       before { download.save(test_file_name) }
 
       it 'has the correct processor' do
-        upload = SampleManifestExcel::Upload::Base.new(file: test_file, column_list: plate_columns, start_row: 9)
-        expect(upload.processor).not_to be_nil
         expect(upload.processor).to be_plate
       end
 
       it 'updates all of the data' do
-        upload = SampleManifestExcel::Upload::Base.new(file: test_file, column_list: plate_columns, start_row: 9)
         upload.process(nil)
         expect(upload).to be_processed
       end
+
+      context 'when accessioning is enabled', accessioning_enabled: true do
+        it 'surpresses accessioning' do
+          expect { upload.process(nil) }.not_to change(Delayed::Job, :count)
```

Comment:
Maybe it would be worth a short comment here saying something like:
```
# accessioning is triggered explicitly after upload
```
if that is accurate?